server/controllers/conversation.js

```diff
@@ -0,0 +1,33 @@
+const Conversation = require("../models/Conversation");
+const asyncHandler = require("express-async-handler");
+
+
+// Given two userIDs, create a conversation
+exports.createConversation = asyncHandler(async (req, res) => {
+    const { senderID, receiverID } = req.body;
+    const newConvo = new Conversation({
```

Comment:
With the current logic, multiple conversations can be created between the same two users. I don't think that's what we need? :)